.vib/harbor-jobservice/goss/vars.yaml

```diff
@@ -0,0 +1,8 @@
+binaries:
+  - harbor_jobservice
+  - yq
+directories:
+  - mode: "0755"
```

Comment:
We are not explicitly setting the permissions for this directory, I would remove the `mode` check.